### ISSUE TITLE
DoAbsolute ABSOLUTE.RData does not exist, drop it ERROR

### ISSUE THREAD

**Issue by tanayb001** [ShixiangWang/DoAbsolute#25]
I have copy number segment file generated using DNACopy package and maf file generated using Mutect2 file. Now I want to use DoAbsolute to infer copy number calls and here is my command which is generating these errors:

```
> DoAbsolute("p1_seg.txt", Maf = "IITK-P01.maf", sigma.p = 0, max.sigma.h = 0.2, min.ploidy = 0.5,
+   max.ploidy = 10, primary.disease = "PRAD", platform = "Illumina_WES",
+   temp.dir = ("/home/iit/Documents/ABSOLUTE/tmp"), clean.temp = FALSE,
+   results.dir = getwd(), max.as.seg.count = 1500,  max.non.clonal = 0.05,  max.neg.genome = 0.005,  
+   copy.num.type = "total",  min.mut.af = 0.1,  min.no.mut = 5,
+   verbose = FALSE,  nThread = 1L,  keepAllResult = TRUE,  recover = FALSE)
-> Setting results directory as /home/iit/Documents/ABSOLUTE 
-> Setting temp directory as /home/iit/Documents/ABSOLUTE/tmp 
Directory /home/iit/Documents/ABSOLUTE/tmp/cache does not exists.
----> Error detected, see log for more details.
--> Files in cache directory:
[1] "tmp"
Warning in DoAbsolute("p1_seg.txt", Maf = "IITK-P01.maf", sigma.p = 0, max.sigma.h = 0.2,  :
  --> Result file /home/iit/Documents/ABSOLUTE/tmp/cache/IITK-P01-TD.ABSOLUTE.RData does not exist, drop it.
Error in DoAbsolute("p1_seg.txt", Maf = "IITK-P01.maf", sigma.p = 0, max.sigma.h = 0.2,  : 
  No result file to proceed.
>
```

Please let me know how to solve it.

**Comment by ShixiangWang** [comment]
There should have a `error.log` in your result directory, what's the output?

**Comment by tanayb001** [comment]
No output is generating. The error log file contains this:
### 
Detected error in sample IITK-P01-TD 
Error message: the condition has length > 1 
Where:
if !is.na(res) {
    mode.tab[(i - 1) * n.d + j, ] <- c(res[[1]], res[[2]], res[[3]])
} 
Skipping this sample.
========
The tmp directory has a cache folder and new maf and .seg file.

The segment input file looks like this:
Sample	Chromosome	Start 	End	Num_Probes  	Segment_Mean
IITK-P01-TD	chr1	16739	13392417	8527	0.3464
IITK-P01-TD	chr1	13392517	13395697	5	-1.04
IITK-P01-TD	chr1	13416259	68375421	28515	0.2675

Please let me know what can I do to run the tool.

**Comment by ShixiangWang** [comment]
Are you using the modified ABSOLUTE?

```
path_to_file = system.file("extdata", "ABSOLUTE_1.0.6.tar.gz", package = "DoAbsolute", mustWork = T)
install.packages(path_to_file, repos = NULL, type="source")
```

If yes, you could send me the file and code to run DoAbsolute, debugging may be required.

**Comment by tanayb001** [comment]
I installed both ABSOLUTE and DoAbsolute package. I Absolute I'm not understanding all the step by step process. Also I got error while running CreateReviewObject command. So I move to use DoAbsolute. Yes I installed using 

path_to_file = system.file("extdata", "ABSOLUTE_1.0.6.tar.gz", package = "DoAbsolute", mustWork = T)
install.packages(path_to_file, repos = NULL, type="source")

from https://github.com/ShixiangWang/DoAbsolute#example

How I'll send you the files? I ran the above code mentioned in the issue.
